Modules/Relationships/test/Relationships.Domain.Tests/Tests/Aggregates/Relationships/DecomposeRelationshipTests.cs

```diff
@@ -90,4 +107,21 @@ public void Identity_to_can_only_decompose_once()
         // Assert
         acting.Should().Throw<DomainException>().WithError("error.platform.validation.relationshipRequest.relationshipAlreadyDecomposed");
     }
+
+    [Fact]
+    public void Two_identities_can_enter_into_a_new_relationship_again_after_decomposing()
+    {
+        // Arrange
+        var existingRelationships = new List<Relationship>
+        {
+            CreateDecomposedRelationship(IDENTITY_1, IDENTITY_2)
+        };
+
+        // Act
+        var newRelationship = new Relationship(RELATIONSHIP_TEMPLATE_OF_1, IDENTITY_2, DEVICE_2, null, existingRelationships);
```

Comment:
I'd rather use an empty list instead of null for the content. `null` always looks suspicious in my opinion. :)
```suggestion
        var newRelationship = new Relationship(RELATIONSHIP_TEMPLATE_OF_1, IDENTITY_2, DEVICE_2, [], existingRelationships);
```

Reply:
Changes made. But I have a question then, why are we using `null` in `TestData` file in some places when creating a new `Relationship`? Should I change there also to empty list?

Reply:
> Should I change there also to empty list?

Yeah, why not.

Reply:
Changes made.